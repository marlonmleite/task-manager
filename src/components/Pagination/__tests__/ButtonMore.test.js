import React from 'react'
import { shallow } from 'enzyme'
import ButtonMore from '../styled'

describe('ButtonMore', () => {
  it('should render', () => {
    const btn = shallow(<ButtonMore>more</ButtonMore>)

    expect(btn).toMatchSnapshot()
  })
})
