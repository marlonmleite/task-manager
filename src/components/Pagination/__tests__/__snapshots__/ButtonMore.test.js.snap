// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonMore should render 1`] = `
.c0 {
  font-weight: bold;
  text-transform: uppercase;
  box-shadow: 0 0 4px rgba(0,0,0,.14),0 4px 8px rgba(0,0,0,.28);
}

<Button
  className="c0"
  color="primary"
  loading={false}
  onClick={null}
  type="button"
>
  more
</Button>
`;
